javascript/login.js

```diff
@@ -0,0 +1,95 @@
+const emailInput = document.querySelector('#email-form');
+const emailSpan = document.createElement('span');
+const passwordInput = document.querySelector('#password-form');
+const passwordSpan = document.createElement('span');
+const eyeButtonImage = document.querySelector('.eye-image');
+const loginButton = document.querySelector('.login-button')
+
+
+
+/*--이메일 형식확인하는 함수--*/
+function isValidEmail(email) {
+  const emailRegex = /^[a-zA-Z0-9._%+-]+@[a-zA-Z0-9.-]+\.[a-zA-Z]{2,}$/;
+  return emailRegex.test(email);
+}
+
+/*--이메일 input에 정확하지 않은 값을 넣고 focusout한 경우--*/
+function emailInputAdd (e) {
+  if (e.target.value.length === 0) {
+    emailSpan.textContent = "이메일을 입력해주세요.";
+  } else if (!isValidEmail(e.target.value) && (e.target.value.length >= 1)) {
+    emailSpan.textContent = "올바른 이메일 주소가 아닙니다.";
+  } else {
+    emailSpan.textContent = '';
+  }
+
+  if (emailSpan.textContent) {
+    emailInput.classList.add('InputEvent');
+  }
+
+  emailSpan.classList.add('inputSpan');
+  emailInput.after(emailSpan);
+}
+
+/*--이메일 input에 focusin한 경우 초기화시켜주기--*/
+function emailInputDelete (e) {
```

Comment:
위에 함수명에 대한 코멘트는 별도로 참고해주시면 좋을 것 같고, 아래와 같이 띄어쓰기를 삭제해야 할 것 같아요! 
```suggestion
function emailInputDelete(e) {
```